<launch>
    <include file="$(find diff_drive_gazebo)/launch/main.launch" />
    <node name="diff_motor_sim" pkg="simulation_control" type="motorspeed_sim_diff.py" />
    <node name="odom_sim" pkg="simulation_control" type="controls.py" output="screen" />
</launch>